middleware.js

```diff
@@ -57,8 +57,12 @@ module.exports = function(options) {
     options = { src: options };
   }
 
+  var sassMiddlewareError = null;
+
   // This function will be called if something goes wrong
-  var error = options.error || function() { };
+  var error = options.error || function(err) {
+    sassMiddlewareError = err;
```

Comment:
If the intention is to pass `errorObject || null` to the next middleware `sassMiddlewareError` seems unnecessary. This fallback function will transparently return whatever node-sass is returning; `null` in case of success. 

Otherwise, your approach is best suited if there is a use-case where you want to use `var sassMiddlewareError` at some later point.